core/lib/libtomcrypt/ecc.c

```diff
@@ -358,43 +359,134 @@ static const struct crypto_ecc_public_ops ecc_public_key_ops = {
 	.verify = &ltc_ecc_verify,
 };
 
+static const struct crypto_ecc_keypair_ops sm2_dsa_keypair_ops = {
+	.generate = &ltc_ecc_generate_keypair,
+	.sign = &ltc_sm2_dsa_sign,
+	.shared_secret = NULL,
```

Comment:
no need to explicitly set this to `NULL`.

Reply:
ok